ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/LedgerApiTestTool.scala

```diff
@@ -93,6 +97,7 @@ object LedgerApiTestTool {
       else config.included
 
     val testsToRun = Tests.all.filterKeys(included -- config.excluded)
+    val performanceTestsToRun = Tests.performanceTests.filterKeys(config.performanceTests)
```

Comment:
If you use `--perf-test` alongside `--include` or `--all-tests`, some of the tests you explicitly specified won't get run. Might wanna raise an error in this case.